nano/node/confirmation_solicitor.cpp

```diff
@@ -1,61 +1,77 @@
 #include <nano/node/confirmation_solicitor.hpp>
 #include <nano/node/election.hpp>
-#include <nano/node/node.hpp>
 
-nano::confirmation_solicitor::confirmation_solicitor (nano::node & node_a) :
-node (node_a)
+using namespace std::chrono_literals;
+
+nano::confirmation_solicitor::confirmation_solicitor (nano::network & network_a, nano::network_constants const & params_a) :
+max_confirm_req_batches (params_a.is_test_network () ? 1 : 20),
+max_block_broadcasts (params_a.is_test_network () ? 4 : 30),
```

Comment:
Could remove `network_constants` dependency and pass in `bool is_test_network` instead, but I don't think it's that important. Haven't tested it, but otherwise LGTM